src/lib/templateengine.js

```diff
@@ -1321,6 +1336,8 @@ function TemplateEngine( undefined ) {
     
     if (page_id.match(/^id_[0-9_]*$/) == null) {
       // find Page-ID by name
+      // decode html code (e.g. like &apos; => ')
+      page_id = $("<textarea/>").html(page_id).val();
```

Comment:
I stumbled upon the problem that a pagejump named "Child's room" didn´t work. In the config it must be written like "Child&apos;s Room" and because of that the scrollToPage function does not find the matching page.
There is not html_(de|en)code functions in javascript, and after some googling the jquery-way was the only solution i found. So you need a complete html escaping method here to make sure everything works. Any suggestions?